docs/src/styles/main.scss

```diff
@@ -62,6 +62,14 @@ figcaption.header {
   padding: 0.5em;
 }
 
+@media screen and (max-width: 1024px) {
+  .navbar-item {
+    margin-top: 0.3em;
+    display: flex !important;
```

Comment:
using `!important` declaratives is not a good practice and reflects misplaced CSS elsewhere. We should try to avoid this.

Reply:
fixed.